main.md

```diff
@@ -824,15 +833,15 @@ the `address` property:
 {#example-simple_structured-release-payload}
 ```json
 {
-  "nonce": "2GLC42sKQveCfGfryNRN9w",
-  "aud": "https://example.com/verifier",
+  "nonce": "_XYz2toJ9Qj8bcRM-_DeBA",
+  "aud": "https://example.com/issuer",
```

Comment:
audience of the release is verifier, right?
```suggestion
  "aud": "https://example.com/verifier",
```

Reply:
Yes, should be the verifier!

Reply:
mmhh... It seems coming from the legacy code. Thank you, I'm doing a general check for this

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/6a7dbc6bbec4ad807ad2b1f1107375a66d3cfecf